### PR TITLE
Api for getting all and selected salary slip(s) of an employee

### PR DESCRIPTION
**Pull request by anjanaNewput**
@niteshthakur5 
Please review and merge.

### CODE REVIEW

File: backend/routes/routes.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ module.exports = function(app, router) {
 
   // Routes for salary slip
   router.post('/admin/upload', validate.validateAdminToken, salarySlipController.uploadSingleFile);
+  router.get('/employee/salaryslip', validate.validateAdminToken, salarySlipController.getEmployeeSalarySlip);
```
Comment (niteshthakur5) [inline backend/routes/routes.js]: Make it '/employee/salary_slips'

File: backend/services/salary_slip_service.js [hunk 2]
```diff
@@ -19,65 +21,81 @@ var uploadSingle = multer({ //multer settings
...
 },
 }).single('file');
 
 var service = {};
 
-service['uploadSingle'] = function(req, res, callback) {
+service['uploadSingle'] = function(request, res, callback) {
   try {
-    uploadSingle(req,res,function(err){
+    uploadSingle(request,res,function(err){
```
Comment (niteshthakur5) [inline backend/services/salary_slip_service.js]: give whitespace after ,

File: backend/rest_controllers/salary_slip_controller.js [hunk 3]
```diff
@@ -20,4 +21,19 @@ api['uploadSingleFile'] = function(req, res) {
   }
 };
 
+api['getEmployeeSalarySlip'] = function(req, res) {
```
Comment (niteshthakur5) [inline backend/rest_controllers/salary_slip_controller.js]: Change it to getEmployeeSalarySlips if it's returning multiple salary slips.

**Comment by anjanaNewput** [comment]
@niteshthakur5 
I have made requested changes, Please merge.

File: backend/routes/routes.js [hunk 4]
```diff
@@ -35,6 +35,7 @@ module.exports = function(app, router) {
 
   // Routes for salary slip
   router.post('/admin/upload', validate.validateAdminToken, salarySlipController.uploadSingleFile);
+  router.get('/employee/salary_slips', validate.validateAdminToken, salarySlipController.getEmployeeSalarySlip);
```
Comment (niteshthakur5) [inline backend/routes/routes.js]: I think you need to change getEmployeeSalarySlip here too.

**Pull request merged**